app/features/receive/cashu-token-swap-repository.ts

```diff
@@ -200,6 +243,35 @@ export class CashuTokenSwapRepository {
     );
   }
 
+  /**
+   * Gets a token swap by token hash.
+   * @returns The token swap.
+   * @throws Error if the token swap does not exist.
+   */
+  async get(
+    tokenHash: string,
+    options?: Options,
+  ): Promise<CashuTokenSwap | null> {
+    const query = this.db
+      .from('cashu_token_swaps')
+      .select()
+      .eq('token_hash', tokenHash);
```

Comment:
use id should also be specified here

also we should use single (check how gets are done in other repositories)